### DISCUSSION TITLE
Mantine & Astro Nano Stores

### DISCUSSION THREAD

**Discussion by joshxyzhimself** [mantinedev/mantine#6720] [Q&A]
Hi all, this is probably a dumb question so please bear with me.

At [Mantine - Can I use Mantine with Astro](https://help.mantine.dev/q/can-i-use-mantine-with-astro) it quotes the following from [Astro - Share state between Islands](https://docs.astro.build/en/recipes/sharing-state-islands/):

> UI frameworks like React or Vue may encourage “context” providers for other components to consume. But when partially hydrating components within Astro or Markdown, you can’t use these context wrappers.

However from [Astro - Share state between Islands](https://docs.astro.build/en/recipes/sharing-state-islands/) I noticed there is a bottom part that says:

> Astro recommends a different solution for shared client-side storage: [Nano Stores](https://github.com/nanostores/nanostores).

So the question is.. could Nano Stores make Mantine work with Astro?

Just curious if anyone has tried or has insights.

Related threads:

- https://github.com/orgs/mantinedev/discussions/1771
- https://github.com/orgs/mantinedev/discussions/4462
- https://github.com/orgs/mantinedev/discussions/3171

**Comment by rtivital** [comment]
No, as far as I know, it is not possible.
